option/internaloption/internaloption.go

```diff
@@ -94,3 +94,15 @@ func (w withDefaultScopes) Apply(o *internal.DialSettings) {
 	o.DefaultScopes = make([]string, len(w))
 	copy(o.DefaultScopes, w)
 }
+
+// EnableJwtWithScope returns a ClientOption that specifies if scope can be used
+// with self-signed JWT.
+func EnableJwtWithScope(enableScope bool) option.ClientOption {
```

Comment:
What I meant by a previous comment was that this does not need to take a bool. If passed it can be assumed true.